package.json

```diff
@@ -11,9 +11,10 @@
 		"lint": "tslint '*/*/src/**/*.ts' '*/*/test/**/*.ts'",
 		"build": "lerna run build",
 		"watch": "lerna run watch",
-		"prep-release": "./scripts/prepareRelease.sh",
-		"validate-release": "./scripts/validateRelease.sh",
-		"publishCanary": "lerna publish --canary",
+		"release:prep": "./scripts/prepareRelease.sh",
+    "release:validate": "./scripts/validateRelease.sh",
```

Comment:
Fix extra space lines 14-15 🔍

Reply:
Actually not sure why this is tab-indented lol